### ISSUE TITLE
Make conversation with designer about buttons in the sidebar

### ISSUE THREAD

**Issue by GGO-web** [GGO-web/Unimay#63]
There are two buttons for search and for menu, I don't think we have a next steps design for those buttons

**Comment by alexey-sevastynov** [comment]
> There are two buttons for search and for menu, I don't think we have a next steps design for those buttons

I don’t understand what I need to ask the designer

**Comment by alexey-sevastynov** [comment]
1. logo link to home page, I understand.
2. I need to understand what these buttons does?? 

![image](https://github.com/GGO-web/Unimay/assets/97807120/1535a573-f65b-4997-821b-2b08f15c6e0b)

**Comment by GGO-web** [comment]
Yes, your last comment is right. We do not know why do we have this buttons and what they actually do
